cacti/tests/conftest.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+from copy import deepcopy
+
+from datadog_checks.dev import docker_run
+import docker
+from datadog_checks.dev.subprocess import run_command
+from datadog_checks.cacti import CactiCheck
+
+from .common import (
+    HERE,
+    HOST,
+    INSTANCE_INTEGRATION
+)
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    env = {'HOST': HOST}
+    with docker_run(
+        compose_file=os.path.join(HERE, "compose", "docker-compose.yaml"),
+        env_vars=env,
+    ):
+        yield INSTANCE_INTEGRATION
+
+
+@pytest.fixture
+def check():
+    container = _get_container()
+    commands = ["apt-get update",
+                "apt-get install rrdtool librrd-dev libpython-dev build-essential -y",
+                "pip install rrdtool"]
+    for command in commands:
+        exit_code, output = container.exec_run(command)
+        if exit_code != 0:
+            raise Exception(output)
+    return CactiCheck('mapreduce', {}, {})
+
+
+def _get_container():
+    docker_ps = [
+        'docker',
+        'ps',
+        '-aqf',
+        ''"name=dd_cacti_py.*"''
+    ]
+    container_id = run_command(docker_ps, capture='out', check=True).stdout.rstrip()
+    client = docker.client.from_env()
```

Comment:
Looks like test fail because we do not have docker already in the python environment. datadog_checks currently uses raw os commands to interact with docker. Have you considered getting rid of the docker library and maybe make/use something generic inside `datadog_checks_dev`? This might be more work than we want to spend here though.

Reply:
I should be able to add the dependency somewhere. requirements-dev.txt?

Reply:
Didn't know we had this. Nice 👍